CSharp/BatchExplorer/Helpers/ResourceFileStringParser.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.BatchExplorer.Models;
+
+namespace Microsoft.Azure.BatchExplorer.Helpers
+{
+    public static class ResourceFileStringParser
+    {
+        private static readonly string[] FileDelimiter = new[] { ";", Environment.NewLine };
```

Comment:
I'm not the biggest fan on this approach since it require the user to type in a ton of stuff, but I guess it's OK for now. I worked on some other UI to use the FilePicker instead and I can integrate that in later.